Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -1915,7 +1915,7 @@ def renameHLTprocessInSequence(self,sequence,proc=None,HLTprocess='HLT'):
 
 
     def expandMapping(self,seqList,mapping,index=None):
-        maxLevel=20
+        maxLevel=25
```

Comment:
@jfernan2 I understand that this is needed due to a deeper level of configuration fragments nesting

Reply:
Correct